app/views/users/show.html.erb

```diff
@@ -10,21 +14,13 @@
 </section>
 
 <section>
-  <h2>Recent Posts</h2>
   <div>
-    <h3>Post 1</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-  </div>
-  <div>
-    <h3>Post 2</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-  </div>
-  <div>
-    <h3>Post 3</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
+    <% @user.posts.each do |post| %>
+      <h3>
+      <%= link_to link_to post.title, user_post_path(@user.id, post.id) %>
+      </h3>
+      <p><%= post.text %></p>
+    <% end %>
   </div>
   <button>See all posts</button>
```

Comment:
- [ ] Clicking on `see all posts` suppose to opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png), So please fix that by using `link_to` or `button_to` method.